py/requirements.txt

```diff
@@ -1,2 +1,4 @@
 oauth2client>=4.0.0
 pandas>=0.17.1,<0.21
+ipython>=5.7.0
```

Comment:
We should test out installing the client lib in Python 2 and 3 to make sure this works.

Reply:
Here's what I was able to find out: this install appears to work fine for both python2 and python3. However (and AFAiCT this is an existing issue), I can't import the concepts package into a py2 notebook, that fails with:

```
ImportError: No module named html
```

That appears to be a problem with or without this requirements.txt change. I'm not sure that bug is worth fixing since we were close to turning off py2.7 anyways.

Reply:
Huh, wonder why I didn't see this before. 

Anyway, I agree; let's proceed with the change.